pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -268,12 +268,19 @@ func (r *InferenceServiceReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	}
 	// Handle InferenceService status updates based on the force stop annotation.
 	// If true, transition the service to a stopped and unready state; otherwise, ensure it's not marked as stopped.
+	existingStoppedCondition := isvc.Status.GetCondition(v1beta1.Stopped)
+	if existingStoppedCondition == nil {
+		defaultStoppedCondition := &apis.Condition{
+			Type:   v1beta1.Stopped,
+			Status: corev1.ConditionFalse,
+		}
+		isvc.Status.SetCondition(v1beta1.Stopped, defaultStoppedCondition)
+	}
+	existingStoppedCondition = isvc.Status.GetCondition(v1beta1.Stopped)
 	if forceStopRuntime {
-		// Exit early if we have already set the status to stopped
-		existingStoppedCondition := isvc.Status.GetCondition(v1beta1.Stopped)
-		if existingStoppedCondition != nil && existingStoppedCondition.Status == corev1.ConditionTrue {
-			// TODO: Set condition to stoppING
```

Comment:
Is it intentional to remove TODO?

Reply:
yes, adding the stopp**ing** condition is a low priority for us and the TODO comment was just adding confusion for reviewers